honeybee_radiance_folder/folder.py

```diff
@@ -228,425 +354,184 @@ def daylight_matrix_files(self, aperture, receiver=None):
 
-    @property
-    def sky_folder(self):
-        """Sky files folder."""
-        return self.SKY
-
-    def sky_files(self, rel_path=True):
-        """List of sky files.
-        
-        Args:
-            rel_path: A boolean to indicate if folders should be returned as relative
-                paths in relation to root folder.
-        """
-        return self._find_files(self.IES, '.sky', rel_path)
-
-    @property
-    def sun_folder(self):
-        """Sunpath and sun source folder."""
-        return self.SUN
-
-    def sun_files(self, rel_path=True):
-        """List of sun files.
-        
-        Args:
-            rel_path: A boolean to indicate if folders should be returned as relative
-                paths in relation to root folder.
-        """
-        return self._find_files(self.SUN, '.sun', rel_path)
-
-    @property
-    def grid_folder(self):
-        """Sensor grids folder."""
-        return self.GRID
+    def dynamic_scene(self, indoor=False, reload=False):
+        """List of dynamic non-aperture geometries.
 
-    def grid_files(self, rel_path=True):
-        """List of sensor grid files.
-        
         Args:
-            rel_path: A boolean to indicate if folders should be returned as relative
-                paths in relation to root folder.
+            indoor (bool): A boolean to indicate if indoor dynamic scene files should be
+                returned (default: False).
+            reload (bool): Dynamic geometries are loaded the first time this
+                method is called. To reload the files set reload to True.
         """
-        return self._find_files(self.GRID, '.pts', rel_path)
-
-    @property
-    def view_folder(self):
-        """View files folder."""
-        return self.VIEW
+        if reload or self._dynamic_scene_load:
+            self._load_dynamic_scene()
 
-    def view_files(self, rel_path=True):
-        """List of view files.
-        
-        Args:
-            rel_path: A boolean to indicate if folders should be returned as relative
-                paths in relation to root folder.
-        """
-        return self._find_files(self.VIEW, '.vf', rel_path)
+        return self._dynamic_scene_indoor if indoor \
+            else self._dynamic_scene
 
-    def write(self, overwrite=True):
-        """Write model folder.
+    def write(self, folder_type=0, cfg=None, overwrite=False):
+        """Write an empty model folder.
 
         Args:
+            folder_type: An integer between 0-2.
+                * 0: grid-based
+                * 1: view-based
+                * 2: includes both views and grids
+            cfg (dict): A dictionary with folder name as key and True or False as
+                value. You can pre-defined from ``config`` module. Default is a
+                grid-based ray-tracing folder.
             overwrite (bool): Set to True to overwrite the folder is it already exist.
-        """
```

Comment:
Should be:
`overwrite (bool): Set to True to overwrite the folder is it already exists. (Default: False).`

FYI, this option to overwrite was not working when I tried to use it in honeybee-radiance. I had to use the `nukedir` method from ladybug in order to be able to overwrite things. So I would recommend writing a test for this.

Reply:
Changed it to nukedir